pkgs/development/python-modules/dodgy/default.nix

```diff
@@ -0,0 +1,33 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, isPy3k
+, lib
+}:
+
+buildPythonPackage rec {
+  pname = "dodgy";
+  version = "0.2.1";
+  disabled = !isPy3k;
+
+  src = fetchFromGitHub {
+    owner = "landscapeio";
+    repo = pname;
+    # There is no tag for this version, so we'll be using the commit instead
+    rev = "7955edb5b0b8ab23c6cfc365e72d61cc1adedc11";
```

Comment:
upstream really should have some principled release structure to their project :(

Reply:
now it have an structure, they tagged their latest release

https://github.com/landscapeio/dodgy/issues/19